File: easy/3516_find_closest_person/solution.cpp
using namespace std;

#include <cstdlib>

class Solution {
public:
  int findClosest(int x, int y, int z) {
    const int xz = abs(x - z);
    const int yz = abs(y - z);

    if (xz == yz)
      return 0;
    else if (yz > xz)
      return 1;
    else
      return 2;
  }
};

File: easy/3516_find_closest_person/solution.py
class Solution:
    def findClosest(self, x: int, y: int, z: int) -> int:
        xz = abs(x - z)
        yz = abs(y - z)

        if xz == yz:
            return 0
        elif yz > xz:
            return 1
        else:
            return 2
